handlers/mempoolView.go

```diff
@@ -47,7 +45,7 @@ func formatToTable(content *types.RawMempoolResponse) *types.DataTableResponse {
 		for _, tx := range pendingData {
 			dataTable.Data = append(dataTable.Data, []any{
 				// TODO: link to tx page once it has a mempool view implemented
```

Comment:
This comment became outdated because of your fix, please remove it :grin:

Reply:
removed ;-)